hybrasyl/Scripting/HybrasylMonster.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Hybrasyl.Objects;
+using MoonSharp.Interpreter;
+using System;
+using System.Collections.Generic;
+using System.Security.Permissions;
+using System.Text;
+
+namespace Hybrasyl.Scripting
+{
+    [MoonSharpUserData]
```

Comment:
This type should be added to `Script.GetUserDataValue`

Reply:
updated